scripts/configure-csrf-protection.groovy

```diff
@@ -18,6 +18,7 @@
  */
 
 import hudson.security.csrf.DefaultCrumbIssuer
+import jenkins.model.*
```

Comment:
Thanks for your contribution.  Instead of wild card can you import the specific class?  e.g.

```
import jenkins.model.Jenkins
```

I prefer explicit imports because it allows new users of Jenkins to know where to look to learn more about classes.  This helps new users who are studying the code.